src/tools/genpolicy/tests/policy/testdata/state/execprocessdeployment/testcases.json

```diff
@@ -0,0 +1,383 @@
+[
+  {
+    "allowed": true,
+    "description": "",
```

Comment:
Could you add a description to the test cases added here?

Reply:
Whoops, yes, sorry